src/util/Components/Forms/CheckList.tsx

```diff
@@ -21,13 +24,42 @@ const CheckList = (props: PropsType) => {
   if (!inactiveClass) inactiveClass = "btn-secondary active";
   if (!activeClass) activeClass = "btn-success active";
 
+  const selectAll = () => {
+    let newSet:string;
+    props.onClickSet(getSet(newSet, props.items));
```

Comment:
getSet method wasn't created for this purpose. 
```suggestion
    props.onClickSet(new Set(props.items));
```